### ISSUE TITLE
Order @param annotations in the order they are defined in the method

### ISSUE THREAD

**Issue by ntzm** [PHP-CS-Fixer/PHP-CS-Fixer#3161]
```php
/**
 * @param string $foo
 * @param string $bar
 */
function baz(string $bar, string $foo) { }
```

should be turned into

```php
/**
 * @param string $bar
 * @param string $foo
 */
function baz(string $bar, string $foo) { }
```

**Comment by grubersjoe** [comment]
I am also highly interested in this feature.

A basic question: Would this be implemented in a new `PhpdocParamOrderFixer` class or could this be integrated into the existing `PhpdocOrderFixer`? I'm thinking about creating a MR.

**Comment by julienfalque** [comment]
Great to hear @grubersjoe! :)

IMO this should be a dedicated fixer, `PhpdocParamOrderFixer` would be fine.

**Comment by Wirone** [comment]
Implemented in #3909 and will be available in the upcoming release 🙂.
